spec/features/staff/users_can_create_a_planned_disbursement_spec.rb

```diff
@@ -54,6 +54,23 @@
       end
     end
 
+    scenario "the planned disbursement is associated with the currently active report" do
+      fund = create(:fund_activity)
+      programme = create(:programme_activity, parent: fund)
+      project = create(:project_activity, :with_report, organisation: user.organisation, parent: programme)
+
+      visit activities_path
+
+      click_on(project.title)
+      click_on(I18n.t("page_content.planned_disbursements.button.create"))
+
+      fill_in_planned_disbursement_form
+
+      planned_disbursement = PlannedDisbursement.last
+      report = Report.find_by(fund: fund, organisation: project.organisation)
```

Comment:
Should the test explicitly create the report we expect the planned disbursement to be associated with?

Reply:
Yeah that makes more sense 👍